ironfish/src/node.ts

```diff
@@ -351,6 +357,8 @@ export class IronfishNode {
 
     await this.memPool.start()
 
+    this.assetsVerifier.start()
```

Comment:
It looks like we don't call stop() below on line 376.

Reply:
Uh oh, good catch